src/fides/api/ops/service/connectors/saas/connector_registry_service.py

```diff
@@ -85,9 +131,10 @@ class ConnectorRegistry:
     def get_instance(cls) -> "ConnectorRegistry":
         if cls._instance is None:
             cls._instance = cls()
-            cls._instance._templates = (
-                FileConnectorTemplateLoader().get_connector_templates()
-            )
+            cls._instance._templates = {
+                **FileConnectorTemplateLoader().get_connector_templates(),
+                **CustomConnectorTemplateLoader().get_connector_templates(),
```

Comment:
Adding new loader

Reply:
Connector templates defined in the database will override connectors bundled with Fides (by design)